src/plugins/discover/public/application/main/components/pattern_analysis/pattern_analysis_table.tsx

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { useEffect, useState, useMemo } from 'react';
+import { METRIC_TYPE, UiCounterMetricType } from '@kbn/analytics';
+import { type EmbeddablePatternAnalysisInput } from '@kbn/aiops-log-pattern-analysis/embeddable';
+import { pick } from 'lodash';
+import { LogCategorizationEmbeddableProps } from '@kbn/aiops-plugin/public/components/log_categorization/log_categorization_for_embeddable/log_categorization_for_embeddable';
```

Comment:
nit: import type

Reply:
Updated in https://github.com/elastic/kibana/pull/178916/commits/1aaa89fc3a28c496c06e84f5335fd50ae31732d2